### ISSUE TITLE
No API for /user/projects/<project_id>/transitive_dependencies ?

### ISSUE THREAD

**Issue by bootstraponline** [versioneye/veye#49]
I'm not seeing a public API to get transitive dependencies.

**Comment by timgluz** [comment]
Yes, because VersionEye/API does not have this endpoint yet; but it could be added easily if there's enough interest from the community.

 could you please open another ticket on the [Versioneye/API](https://github.com/versioneye/versioneye-api/issues) repo and describe your use-case, expectations (aka is pagination ok, do you need to specify filtering or depth, etc)

**Comment by bootstraponline** [comment]
Ok, I opened a ticket there.
